olp-cpp-sdk-dataservice-read/src/StreamLayerClient.cpp

```diff
@@ -82,6 +82,16 @@ client::CancellableFuture<PollResponse> StreamLayerClient::Poll() {
   return impl_->Poll();
 }
 
+client::CancellationToken StreamLayerClient::Seek(
+    SeekRequest request, SeekResponseCallback callback) {
+  return impl_->Seek(request, callback);
```

Comment:
There is a copying of the `request`. Would it be better to use const-ref, as the `impl_->Seek` method only introspects the request and doesn't store it?